.github/workflows/release.yaml

```diff
@@ -0,0 +1,57 @@
+name: Build and Push Database Exporter
+
+on:
+  push:
+    tags:
+      - "*.*.*-*"
+
+env:
+  UBUNTU_VERSION: '22.04'
+  RUST_IMAGE_TAG: '1.73.0'
+  S3_BUCKET: s3://eur2ccd.concordium.com
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  release-euro2ccd:
+    runs-on: ubuntu-latest
+    environment: release
+    steps:
+      - name: Get the eur2ccd version
+        env:
+          TAG: ${{ github.ref_name }}
+        run: echo "EUR2CCD_VERSION=${TAG%%-*}" >> $GITHUB_ENV
+
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.ref_name }}
+          submodules: recursive
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-region: "eu-west-1"
+          role-to-assume: "arn:aws:iam::192549843005:role/github_eur2ccd"
+          role-session-name: ReleaseEur2ccdSession
+
+      - name: Check built packages
+        run: |
+          echo S3_OBJ=$(aws s3 ls s3://eur2ccd.concordium.com/concordium-eur2ccd_${{ github.ref_name }}_amd64.deb) >> $GITHUB_ENV
+
+      - name: Build
+        if: ${{ env.S3_OBJ == '' }}
+        run: |
+          docker build -t ccd-service-builder --build-arg ubuntu_version=${{ env.UBUNTU_VERSION }} --build-arg rust_image_tag=${{ env.RUST_IMAGE_TAG }} -f scripts/debian-package/deb.Dockerfile .
+          set -euxo pipefail
+          id=$(docker create ccd-service-builder)
+          docker cp $id:/build/pkg-root/ eur2ccd-deb
+          docker rm $id
```

Comment:
Not really needed but sure

Reply:
This was just copy pasted from jenkins, I will remove it.